app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,17 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="row.canDeploy()"
+                ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="!(row.canDeploy())"
+                class="disabled-link"
+                disabled>
```

Comment:
yeah thats true.  I'll add the `role="button"`, that should be there as well.  I don't think we are using tabindex regularly though, correct?